index.css

```
@@ -163,14 +198,120 @@ footer {
   color: var(--gray200);
 }
 
-@media (max-width:1920px) {
+@media (min-width:1200px) and (max-width:1920px) {
   .gnb {
     padding: 9px 200px
   }
 }
 
-@media (max-width:1280px) {
-  html {
-    font-size: 12px;
+/* (min-width:768px) and  */
+@media (max-width:1199px) {
+  .gnb {
+    padding: 9px 24px
+  }
+
+  .home-top-section img,
+  .home-bottom-section img {
+    max-width: 100%;
+  }
+
+  .home-top-section {
+    height: 105vw;
+  }
+
+  .home-top {
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .home-top-content {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .home-top-content p {
+    width: 100%;
+  }
+
+  .home-section {
+    gap: 52px;
+    padding-bottom: 56px;
+  }
+
+  .home-section img {
+    width: 100%;
+  }
+
+  .home {
+    height: auto;
+  }
+
+  .home-content {
+    background-color: #FFFFFF;
+    flex-direction: column;
+    gap: 24px;
+    width: 100%;
+  }
+
+  .home:nth-child(2) .home-content {
+    flex-direction: column-reverse;
+    align-items: flex-end;
+  }
+
+  .home-content-2 {
+    text-align: end;
+    align-items: flex-end;
+    margin-right: 0;
+    margin-left: 12px;
+  }
+
+  .home-title {
+    width: 100%;
+  }
+
+  .home-bottom-section {
+    background-color: #CFE5FF;
+    height: 125vw;
+  }
+
+  .home-bottom {
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+    text-align: center;
+  }
+
+  .footer-content {
+    padding: 32px 10%;
+  }
+}
+
+@media (min-width:375px) and (max-width:767px) {
```

💊 제안  
min-width가 조건에 걸려있어 375px 미만으로 화면이 작아지게 되면 PC 스타일이 적용되네요~  
body에 min-width: 375px를 추가하셔서 해당 사이즈 미만으로 작아지게 되면 가로 스크롤이 생기게 해주시거나  
아니라면 min-width를 조건문에서 빼주시는 것을 추천드려요~

```suggestion
@media (max-width:767px) {
```